/**:
  ros__parameters:
    step_time_period_sec: 0.05
    non_critical_log_time_period_ms: 2000
    use_perception: false
    use_params: true
    start_in_pit: false
    num_start_laps: 0
    cte_threshold: 4.0
    cte_merge_threshold: 1.5
    lap_threshold: 20.0
    passing_sf_threshold: 100.0
    back_straight:
      start_percentage: 40.0
      end_percentage: 58.0
    manual_overrides:
      auto_enabled: false
      limit_auto_throttle: false
    timeout:
      manual:
        localization_timeout: 0.5
        low_level_fault_timeout: 0.5
        race_control_timeout: 3.0
        input_manual_command_timeout: 1.0
        opp_car_detections_timeout: 0.5
      autonomous:
        localization_timeout: 0.5
        low_level_fault_timeout: 0.5
        race_control_timeout: 3.0
        input_manual_command_timeout: 3.0
        opp_car_detections_timeout:  0.5
    behavior_tree:
      zmq_max_msgs_per_sec: 100
      groot_publisher_port: 1666
      groot_server_port: 1667
      groot_ip_addr: "*"
    ttls:
      left_ttl_index: 15
      right_ttl_index: 15
      optimal_ttl_index: 15
      race_ttl_index: 15
      pit_ttl_index: 2
    speed_limit:
      pit_lane: 5.0
      pit_road: 10.0
      pit_crawl: 10.0
      yellow: 15.0
      green: 20.0
      max: 50.0
    gaps:
      object_detection: 60.0
      attacker_preparing: 33.0
      attacker_attacking: 18.0
      defender_overtaken: 33.0
      defender_vicinity: 13.0
      attacker_done: 18.0
    speed_modifiers:
      overtaking: [15.0, 15.0]
      speed_threshold_for_overtaking: [ 0.0, 150.0]
      catchup: [15.0, 15.0]
      speed_threshold_for_catchup: [ 0.0, 150.0]
    target_speed_profiles:
      enabled: false
      tsp_name: "hawaii_pursuit"
      tsp_segment_names: ["start", "end"]
      lap_percentages: [0.0, 100.0] 
      speed_profiles: [8.0, 10.0]
    scale_factors:
      enabled: true
      lap_percentages: [0.0, 100.0]
      scale_factors: [100.0, 100.0]
